tests/testthat/test-test-threads.R

```diff
@@ -0,0 +1,222 @@
+context("test_threads(.dry_run=T)")
+
+###################################
+# testing single model submission
+###################################
+
+
+# define constants
```

Comment:
These tests all look good functionally, but at some point we should go back and refactor them (and probably some other ones, for example in the `test-workflow-bbi.R`) to be more self-contained and use setup helpers like [this one](https://github.com/metrumresearchgroup/bbr/blob/a9c8b90432bbeef88ab3a9f15dc1b6a8e22e626a/tests/testthat/setup-workflow-ref.R#L197-L202) that @rymarinelli is adding in #484